### PR TITLE
Add `WSL Integrations` page back on Windows

### LINKED ISSUE #2029: Add `WSL Integration` page back on Windows
I accidentally removed the `WSL Integration` page, which we still need, in #1923. This is for adding it back.

### PR DESCRIPTION
**Pull request by adamkpickering**
Closes #2029.

Does what it says in the title. I have not been able to test this on Windows - maybe whoever reviews this can do so. I think it should work fine, but I could easily have missed something.

### CODE REVIEW

File: src/pages/Integrations.vue
Comment (mook-as) [inline src/pages/Integrations.vue:47]: > ```ts
>   toggleIntegration(name: string, value: boolean) {
>     this.$set(this.busy, name, value);
>     this.$emit('integration-set', name, value);
>   }
> ```

`distro` is a string, `value` is a boolean.

File: src/components/Integration.vue
Comment (mook-as) [inline src/components/Integration.vue:57]: https://github.com/rancher-sandbox/rancher-desktop/blob/732e876b3dc66c571e62688455eead18f8a61fc4/src/utils/paths.ts#L92-L94

This throws an exception. It doesn't make sense on Windows anyway (we're not doing tools here).

File: background.ts
Comment (mook-as) [inline background.ts:492]: Check [`k8smanager.backend`](https://github.com/rancher-sandbox/rancher-desktop/blob/732e876b3dc66c571e62688455eead18f8a61fc4/src/k8s-engine/k8s.ts#L108) instead; that's just a string (and means we don't need to import `WSLBackend` at all, I think).

**Review by mook-as — CHANGES_REQUESTED** [review]

File: background.ts
Comment (adamkpickering) [reply background.ts:492]: image

Typescript complains :cry:

Comment (mook-as) [reply background.ts:492]: Ah, sorry, I didn't realize `listIntegrations` only exists on the WSL backend now. In that case, yes, we need to actually import.

Sorry about that.

File: src/pages/Integrations.vue
Comment (ericpromislow) [inline src/pages/Integrations.vue:47]: I don't this comment applies anymore, although it was in the code that got deleted with types `string` and `boolean` for the two parameters.

**Review by ericpromislow — DISMISSED** [review]
Works fine. Only comment is on an obsolete comment. Also I realize it's not part of this PR, but could you fix the typo at https://github.com/rancher-sandbox/rancher-desktop/blob/c7a685a736f7836f36cb79d8fcf8abfdddd117f5/src/typings/electron-ipc.d.ts#L111 and change `dalog-populate` to `dialog/populate`?

**Comment by mook-as** [comment]
Please don't touch `dalog-populate` — it's been fixed on `main`, so it'll probably just make for a conflict.

**Review by ericpromislow — DISMISSED** [review]
I dismissed my previous review, but am still listed here. So LGTM (now)

File: src/layouts/default.vue
Comment (mook-as) [inline src/layouts/default.vue:102]: I didn't realize `ApplicationSettings` only happened with lima; that might need a better name… but not in this PR.

Ideally we'd call it `/WSLIntegrations` instead, though, to be clearer that it's Windows only.

File: src/pages/Integrations.vue
Comment (mook-as) [inline src/pages/Integrations.vue:16]: I realize you probably just restored this from an old commit, but this should be (because this is already a TypeScript chunk):
```suggestion
      integrations: {} as Record<string, boolean | string>,
```

File: src/pages/Integrations.vue
Comment (mook-as) [inline src/pages/Integrations.vue:22]: This is now irrelevant (the whole page only shows on Windows, so this will always be true).

**Review by mook-as — CHANGES_REQUESTED** [review]

File: background.ts
Comment (adamkpickering) [reply background.ts:496]: Yeah. I just took this whole function and put it inside the `if (k8smanager instanceof WSLBackend) { ... }` block. Silly...

**Comment by adamkpickering** [comment]
Note to self: this will probably require adjustment once #2039 is merged.

File: e2e/pages/nav-page.ts
Comment (mook-as) [inline e2e/pages/nav-page.ts:10]: Nit: alignment is off.
```suggestion
  WSLIntegrations: (page: Page) => new WSLIntegrationsPage(page),
```

**Review by mook-as — APPROVED** [review]
(There's an issue on `main` around `removeLegacySymlinks` that breaks Windows, but that's not the fault of this PR.)

**Review by mook-as — APPROVED** [review]

**Pull request merged**
